src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -149,6 +150,19 @@ describe('sidebar.components.sidebar-content', function() {
     });
   }
 
+  function createSidebarContent(
+    { userid } = { userid: 'acct:person@example.com' }
```

Comment:
Small aside - This can also be written as `{ userid = 'acct:person@example.com ' } = {}`. If you have an object with many keys, its easier to put the defaults on the left side of the main assignment.